cuda_bindings/tests/test_nvjitlink.py

```diff
@@ -5,13 +5,21 @@
 import pytest
 
 from cuda.bindings import nvjitlink
+from cuda import nvrtc

```

Comment:
nit: let's just use the new layout to avoid warnings
```suggestion
from cuda.bindings import nvrtc
```